contracts/pool/concentrated/ConcentratedLiquidityPoolStaker.sol

```diff
@@ -84,7 +84,12 @@ contract ConcentratedLiquidityPoolStaker {
         for (uint256 i; i < incentiveId.length; i++) {
             Incentive memory incentive = incentives[pool][incentiveId[i]];
             Stake storage stake = stakes[positionId][incentiveId[i]];
-            require(stake.secondsGrowthInsideLast == 0, "SUBSCRIBED");
+
```

Comment:
allows BOTH initial subscribes and resubscribes but only when `position.latestAddition > stake.timestamp`